holidays/countries/sierra_leone.py

```diff
@@ -0,0 +1,162 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.islamic import _CustomIslamicHolidays
+from holidays.groups import (
+    ChristianHolidays,
+    InternationalHolidays,
+    IslamicHolidays,
+)
+from holidays.observed_holiday_base import ObservedHolidayBase, SAT_SUN_TO_NEXT_WORKDAY
+
+
+class SierraLeoneIslamicHolidays(_CustomIslamicHolidays):
+    """
+    Sierra Leone Islamic holidays.
+
+    References:
+        * <https://www.timeanddate.com/holidays/sierra-leone/>
+    """
+
+    # Prophet's Birthday
+    MAWLID_DATES = {
+        2018: (11, 21),
+        2019: (11, 10),
+        2020: (10, 29),
+        2021: (10, 18),
+        2022: (10, 8),
+        2023: (9, 27),
+        2024: (9, 15),
+    }
```

Comment:
This class should not be placed here. Look at other countries. And pay attention to the dates style.